### PR TITLE
fix a bug in the build-in-docker script

### PR DESCRIPTION
**Pull request by nunnatsa**
Signed-off-by: Nahshon Unna-Tsameret <nunnatsa@redhat.com>

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: hack/build-in-docker.sh [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
 
 get_image_tag() {
     local current_commit today
     current_commit="$(git rev-parse HEAD)"
     today="$(date +%Y%m%d)"
     echo "v${today}-${current_commit:0:7}"
 }
+
+source hack/common.sh
+main "$@"
```
Comment (tiraboschi) [inline hack/build-in-docker.sh:31]: can you please add a newline?

Comment (nunnatsa) [reply hack/build-in-docker.sh]: Done

**Comment by nunnatsa** [comment]
ci/prow/hco-e2e-upgrade-aws currently runs on OCP 4.5 and it's going to fail due to a side effect in the fix of https://bugzilla.redhat.com/1825330
It will be addressed on operator-framework/operator-lifecycle-manager#1470 on OLM side.
Temporary ignoring it
/override ci/prow/hco-e2e-upgrade-aws

**Comment by tiraboschi** [comment]
ci/prow/hco-e2e-upgrade-aws currently runs on OCP 4.5 and it's going to fail due to a side effect in the fix of https://bugzilla.redhat.com/1825330
It will be addressed on operator-framework/operator-lifecycle-manager#1470 on OLM side.
Temporary ignoring it
/override ci/prow/hco-e2e-upgrade-aws

**Pull request merged**